romfs/lang/fr.json

```diff
@@ -1,196 +1,196 @@
+        "version_text2": "Vos patchs de signature ont été mis à jour pour la version d'HOS ",
+        "install_complete": "Installation complète !",
+        "complete_desc": "Redémarrez votre console pour appliquer",
+        "restart": "Redémarrer",
+        "later": "Je le ferai plus tard.",
+        "extract_failed": "Impossible d'extraire les fichiers !",
+        "restore_failed": "Impossible de restaurer le fichier original patches.ini d'Hekate ! Continuer à désinstaller ?",
+        "uninstall_complete": "Désinstallation complète",
+        "remove_failed": "Impossible de supprimer les patchs de signature",
+        "remove_failed_desc": "Les fichiers ont peut-être été renommés ou supprimés",
+        "generic_error": "Echec de l'installation des patchs de signature !"
     },
     "options": {
         "menu_items": {
-            "ignore_firm": "Ignore minimum firmware version required by titles",
-            "nca_verify": "Verify NCA signatures before installation",
-            "boost_mode": "Enable \"boost mode\" during installations",
-            "ask_delete": "Ask to delete original files after installation",
-            "auto_update": "Check for updates to Awoo Installer automatically",
-            "gay_option": "Remove anime",
-            "sig_url": "Signature patches source URL: ",
-            "language": "Language: ",
-            "check_update": "Check for updates to Awoo Installer",
-            "credits": "Credits"
+            "ignore_firm": "Ignorez la version minimale de firmware requise par les jeux",
+            "nca_verify": "Vérifier les signatures des NCA avant l'installation",
+            "boost_mode": "Activer le  \"boost mode\" pendant l'installation",
+            "ask_delete": "Demande de suppression des fichiers originaux après l'installation",
+            "auto_update": "Vérifier les mises à jour de l'installateur Awoo automatiquement",
+            "gay_option": "Enlever l'anime",
+            "sig_url": "Patchs de signatures URL: ",
+            "language": "Langage: ",
+            "check_update": "Vérifiez les mises à jour de l'installateur Awoo",
+            "credits": "Crédits"
         },
         "nca_warn": {
-            "title": "Warning!",
-            "desc": "Some installable files may contain malicious contents! Only disable this\nfeature if you are absolutely certain the software you will be installing\nis trustworthy!\n\nDo you still want to disable NCA signature verification?",
-            "opt1": "Yes, I want a brick"
+            "title": "Avertissement!",
+            "desc": "Certains fichiers installables peuvent contenir des contenus malveillants ! Ne désactivez cette fonction\nque si vous êtes absolument certain que le logiciel que vous allez installer est\ndigne de confiance !\n\nVoulez-vous toujours désactiver la vérification de la signature des NCA ?"`
```

Comment:
, instead of `

Reply:
Totally. Thank you for taking the time to proofreading me. Sorry =)